repos/spack_repo/builtin/packages/visit/package.py

```diff
@@ -420,3 +420,28 @@ def determine_version(cls, exe):
 
     # see https://github.com/visit-dav/visit/issues/20055
     unresolved_libraries = ["*"]
+
+    def test_smoke_test(self):
+        """Test visit -cli"""
+        spec = self.spec
+
+        if "~python" in spec:
+            raise SkipTest("Package must be installed with +python")
+
+        visit_cmd = Executable(spec["visit"].prefix.bin.visit)
+        testdir = "smoke_test_build"
+        with working_dir(testdir, create=True):
+            script_file_path = "script_file.py"
+            with open(script_file_path, "w") as f:
+                f.write(
+                    """
+                    from visit import *
+                    Launch()
```

Comment:
```suggestion
                    if "AddPlot" in dir():
                         print("VisIt Success")
```
When run with VisIt's CLI visit will already be launched and imported

Reply:
Thanks Cyrus